assets/javascripts/discourse-assign/controllers/assign-user.js

```diff
@@ -13,6 +21,7 @@ export default Controller.extend({
   taskActions: service(),
   autofocus: not("capabilities.touch"),
   assigneeName: or("model.username", "model.group_name"),
+  priorities: PRIORITIES,
```

Comment:
you should set this in init, and have it as `priorities: null` here, this is mostly due to controllers being singletons in Ember.

Reply:
I can see why "assignments" (`this.a = b`) may need to be done on `init`, but should this be the case even if `priorities` should be immutable -- they don't change at all nor by any setting -- and should be the same wherever invoked?